File: CodeIQ-q1220/CodeIQ-q1220.c
i,j;main(){for(i=0;i<25;i++){for(j=0;j<25;j++)(i==j)?printf("%c",'B'+i):printf("A");printf("\n");}}
i;main(j){for(;i<25;printf("\n"),i++)for(j=0;j<25;j++)(i==j)?printf("%c",'B'+i):printf("A");}
i;main(j){for(;i<25;printf("\n"),i++)for(j=0;j<25;j++)printf((i==j)?"%c":"A",'B'+j);}
int i=0,j;for(;i<25;printf("\n"),i++)for(j=0;j<25;j++)printf((i==j)?"%c":"A",'B'+j);
int i=0,j;for(;i++<25;printf("\n"))for(j=1;j<26;)printf((i-j++)?"A":"%c",'A'+j);
int i,j;for(;i++<25;printf("\n"))for(j=0;j++<25;)printf("%c",65+((i-j)?0:j));
int i,j;for(;i++<25;printf("\n"))for(j=0;j++<25;)printf("%c",(i-j)?65:65+j);
int i,j;for(;i++<25;puts(""))for(j=0;j++<25;)printf("%c",(i-j)?65:65+j);
int i,j;for(;i++<25;)for(j=0;j++<26;)printf("%c",26-j?i-j?65:65+j:13);
int i,j;while(i++<25)for(j=0;j++<26;)printf("%c",26-j?i-j?65:65+j:13);
int i,j;for(;i++<25;)for(j=0;j++<26;)putchar(26-j?i-j?65:65+j:13);

File: CodeIQ-q1220/CodeIQ-q1220.rb
j=66;(0..24).each{|i|puts"A"*i+j.chr+"A"*(24-i);j+=1}
j=66;25.times{|i|puts"A"*i+j.chr+"A"*(24-i);j+=1}
j=66;25.times{|i|$><<?A*i+j.chr+?A*(24-i);puts"";j+=1}
25.times{|i|$><<?A*i+(66+i).chr+?A*(24-i);puts""}
A="A"*25;25.times{|i|b=A.dup;b[i]=(i+66).chr;puts b}
25.times{|i|A="A"*25;A[i]=(i+66).chr;puts A}
25.times{|i|puts ?A*i+(66+i).chr+?A*(24-i)}
A="A"*24;25.times{|i|puts A[0,i]+(66+i).chr+A[0,24-i]}

